packages/components/src/components/notification-toast/notification-toast.css

```diff
@@ -156,22 +158,27 @@
   close icon part
 */
 
-.notification-message__icon-close {
-  cursor: pointer;
+.notification-toast__button-close {
   position: absolute;
-  top: 8px;
-  right: 13.5px;
+  top: 10px;
+  right: 7.5px;
+  color: #191919;
```

Comment:
can we use a token for this color?

Reply:
also maybe for top and right?

Reply:
Unfortunately, we don't have tokens for that. Otherwise we would have to change the values